MILESTONE-1.md

```diff
@@ -40,11 +40,11 @@ The authenticator contract address that is used to verify the addresses of voter
 - **next_proposal_nonce** = 1  
 An increasing nonce to make the proposal id unique.
 
-- **proposals** = TBD  
-A mapping of proposals by ids with required information: execution_hash, start, end.
+- **proposals** = { 1: { start: 123.., end: 456.., execution_hash: '0x123...', scores: { 0: 123, 1: 45, 3: 6 }]
+A mapping of proposals by ids with required information: execution_hash, start, end, scores with a counter for the vote For, Against and Abstain.
 
-- **votes** = TBD  
-A mapping of votes by proposal id with required information: address, choice, and voting power.
+- **has_voted** = { 1: [0x123..., 0x456...], 2: [0x789...] }  
```

Comment:
Same as above, this map should be between proposal_id and vote_address to a boolean